src/javascript_bindings.pyx

```diff
@@ -100,10 +100,14 @@ cdef class JavascriptBindings:
             properties = self.properties
             objects = {}
             for objectName in self.objects:
-                methods = {}
-                for methodName in self.objects[objectName]:
-                    methods[methodName] = None
-                objects[objectName] = methods
+                attrs = {}
+                for name, value in self.objects[objectName].items():
+                    if self.IsValueAllowedRecursively(value) and \
+                            name == '__call__' or not name.startswith('_'):
```

Comment:
I think that excluding attributes starting with two underscores is fine, however attrs with single underscore is open for discussion, I think it should be fine to let single underscores be binded.